styles/index.css

```diff
@@ -0,0 +1,93 @@
+body {
+    display: grid;
+    grid-template: 1fr 1fr 1fr 1fr / 3fr .25fr 3fr;
```

Comment:
Stellar use of grid! This looks really good and resizes well to small screen sizes.